config/environments/production.rb

```diff
@@ -72,17 +65,17 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
-
-  # Use a different logger for distributed setups.
-  # require 'syslog/logger'
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
+  # Logging configuration
+  if ENV['LOGSTASH_ENABLE'] == 'true'
+    require './lib/logging'
```

Comment:
I think autoload might have our back here, but not sure?

Reply:
My local environment dies without the explicit require there.